test/NuGet.Core.Tests/NuGet.Configuration.Test/PackageSourceMappingProviderTests.cs

```diff
@@ -46,6 +46,40 @@ public void GetPackageSourceMappingItems_WithOneConfig_ReturnsCorrectPatterns()
             packageSourceMappingSourceItem.Patterns.First().Pattern.Should().Be("stuff");
         }
 
+        [Fact]
+        public void GetPackageSourceMappingItems_WithOneConfig_WithDuplicateKeys_Throws()
```

Comment:
I'm curious how this would behave in a few scenarios (ie, let's add some more tests)


```
user-wide defines nuget.org
solution-wide defines nuget.org
```

```
user-wide definies nuget.org
solution-wide clears
solution-wide defines nuget.org
```

I think the latter one should just work because of the clear.

Reply:
+1 on adding those tests.

Reply:
After some discussions, we conluded that the duplicates will only be detected *inside* the same config. 

We won't be detecting duplicates across different configs correct?

Can oyu please update the description to clarify the behavior?